<ui version="4.0" >
 <class>ImportTool</class>
 <widget class="QDialog" name="ImportTool" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Import Images</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>10</number>
   </property>
   <property name="verticalSpacing" >
    <number>5</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>-1</number>
     </property>
     <property name="leftMargin" >
      <number>10</number>
     </property>
     <property name="rightMargin" >
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Translate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="patternInput" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string/>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="currentIndex" >
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addRegExpButton" >
       <property name="text" >
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removRegExpButton" >
       <property name="text" >
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="leftMargin" >
      <number>10</number>
     </property>
     <property name="rightMargin" >
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Pattern Error:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="patternMessage" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QTableWidget" name="importView" />
     </item>
     <item>
      <widget class="QTableWidget" name="parsedView" >
       <row>
        <property name="text" >
         <string>d</string>
        </property>
       </row>
       <column>
        <property name="text" >
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Angle</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Number</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Sub-Image</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportTool</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportTool</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
